solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -671,6 +671,10 @@ public void load() {
     // Always add $SOLR_HOME/lib to the shared resource loader
     Set<String> libDirs = new LinkedHashSet<>();
     libDirs.add("lib");
+    // Always add $SOLR_TIP/lib to the shared resource loader, to allow loading of i.e. /opt/solr/lib/foo.jar
```

Comment:
I'm further reminded of my earlier code review in which I asked that such stuff be the job of NodeConfig and not CoreContainer.

Reply:
Ok, I did an attempt to move the sharedLib init (both SOLR_HOME/lib and SOLR_TIP/lib into NodeConfig. See latest commit.